test/run/test_stages.py

```diff
@@ -658,3 +658,36 @@ def test_erofs(self):
                 check=True,
             )
             assert "testfile" in r.stdout
+
+    def test_bootc_install_to_fs(self):
+        datadir = self.locate_test_data()
+        testdir = os.path.join(datadir, "stages", "bootc.install-to-fs")
+
+        with self.osbuild as osb, tempfile.TemporaryDirectory(dir="/var/tmp") as outdir:
+            osb.compile_file(os.path.join(testdir, "manifest.json"), exports=["image"], output_dir=outdir)
+
+            image = os.path.join(outdir, "image", "disk.img")
+            assert os.path.isfile(image)
+
+            # check that we have a basic boot/ostree dir layout on /
+            env = os.environ.copy()
+            env["LIBGUESTFS_BACKEND"] = "direct"
+            # check xattrs
+            r = subprocess.run(
+                [
+                    "guestfish",
+                    "-a", image,
+                    "-m", "/dev/sda4:/",
+                    "--",
+                    "ls", "/", ":",
+                    "getxattrs", "/ostree", ":",
```

Comment:
This is OK, but I think the most important thing to check here is actually the label of `/` which should be `root_t`.

But hopefully if this test is passing then it verifies that https://github.com/osbuild/bootc-image-builder/issues/149 is fixed by this?